jenkins/jk-setup.sh

```diff
@@ -133,12 +133,11 @@ elif [[ $COMPILER == *icc* ]]; then
   export CC=`which icc`
   export CXX=`which icc`
   export FC=`which ifort`
-  export ExtraCMakeOptions="${ExtraCMakeOptions} -Dvc=OFF"
 fi
 
 if [[ $ARCH == ppc64le ]]; then
     # The ppc64le build node does not have X11 or GSL installed, and Vc does not support ppc64le
-    export ExtraCMakeOptions="${ExtraCMakeOptions} -Dvc=OFF -Dx11=OFF -Dbuiltin_gsl=ON"
+    export ExtraCMakeOptions="${ExtraCMakeOptions} -Dx11=OFF -Dbuiltin_gsl=ON"
```

Comment:
This is now taken care in ROOT. Vc is disabled there on unsupported platforms, so it's fine to remove the logic from here. I will soon make a new VecCore release that works on ARM and PPC too so that we can enable it on these platforms.